src/libraries/System.Security.Cryptography.Algorithms/src/Resources/Strings.resx

```diff
@@ -162,6 +162,9 @@
   <data name="Cryptography_HashAlgorithmNameNullOrEmpty" xml:space="preserve">
     <value>The hash algorithm name cannot be null or empty.</value>
   </data>
+  <data name="Cryptography_HashSizeInvalid" xml:space="preserve">
+    <value>The specified hash's size is not supported on this platform.</value>
```

Comment:
Since DSACryptoServiceProvider already has this logic, it makes sense to reuse the resource string it uses; vs making up a new one.

Reply:
https://github.com/dotnet/runtime/blob/87f391001eca716a8db896f5c3855d33fe30aca8/src/libraries/System.Security.Cryptography.Csp/src/System/Security/Cryptography/DSACryptoServiceProvider.Windows.cs#L485-L486

https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.Security.Cryptography.Csp/src/Resources/Strings.resx#L149-L151

Reply:
Thanks; the other one was that the Unix DSA CSP's `CreateSignature` wasn't checking the input size, so I moved it (yet another breaking change)